CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## v1.13.0
+
+## v.1.13.0
```

Comment:
Can you remove this line `## v.1.13.0`?

Reply:
a7d132a5c1cef56e8569cf343be17d9ea5ceb100